internal/operator/orbiter/kinds/providers/gce/computesservice.go

```diff
@@ -272,8 +288,16 @@ func (m *machinesService) instances() (map[string][]*instance, error) {
 		return nil, err
 	}
 
-	m.cache.instances = make(map[string][]*instance)
+	if m.cache.instances == nil {
+		m.cache.instances = make(map[string][]*instance)
+	} else {
+		for k := range m.cache.instances {
+			m.cache.instances[k] = nil
+			delete(m.cache.instances, k)
+		}
+	}
```

Comment:
this is never true

Reply:
it can be true, as the cache is only used in this file and the map is only defined in this if. So there is a possibility that the slice is nil if the slice is no initialized while defining the machineService.